crates/nostr/src/event/tag.rs

```diff
@@ -5,10 +5,10 @@ use std::fmt;
 use std::str::FromStr;
 
 use anyhow::{anyhow, Result};
-use bitcoin_hashes::sha256;
 use secp256k1::schnorr::Signature;
 use secp256k1::XOnlyPublicKey;
 use url::Url;
+use crate::Sha256Hash;
 
```

Comment:
Add a line space between `Url` and `Sha256Hash` import.

```rust
use secp256k1::XOnlyPublicKey;
use url::Url;

use crate::Sha256Hash;
```

Reply:
Done